webapp/components/Upload/index.vue

```diff
@@ -41,6 +41,12 @@ export default {
       hover: false,
     }
   },
+  computed: {
+    avatarUrl() {
+      const { avatar } = this.user
+      return avatar && avatar.url
```

Comment:
anybody else feel like using the [optional chaining operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)? :sweat_smile:

it's still experimental technology but I'd like to see if there's a webpack plugin for it already...

Reply:
you want more fun configuring webpack? I got no problem with using it, although the `experimental` part makes me a bit nervous.